docs/roles/developer/contracts/assemblyscript.md

```diff
@@ -0,0 +1,732 @@
+---
+id: assemblyscript
+title: AssemblyScript
+sidebar_label: AssemblyScript
+---
+
+The NEAR platform supports writing contracts in Rust and AssemblyScript.
+
+AssemblyScript is a subset of TypeScript that compiles to Wasm.  See the [official AssemblyScript docs](https://docs.assemblyscript.org) for more details.
+
+This document aims to introduce developers already comfortable with TypeScript to writing AssemblyScript on the NEAR platform.  
+
+If you are not familiar with TypeScript then this [introduction](https://learnxinyminutes.com/docs/typescript/) will be worth a quick look but do keep in mind that AssemblyScript is a *subset of TypeScript* so not all of the features of TypeScript are supported.
```

Comment:
What do we think about linking to:
https://www.typescriptlang.org/docs/handbook/typescript-in-5-minutes.html

Reply:
that's probably a more authoritative resources, yes

Reply:
@mikedotexe can we try the new docs editing workflow for this after merge?  

i mean, normally to fix these kinds of minor link updates is a pain but we've recently added the 
 `[ Edit ]` button to the top right side of all pages in docs. 

this is a perfect candidate.  here are steps for anyone else that stumbles into this (will add to docs as well)

*after we merge this branch to master so we're not clobbering the same file*

0. you need a github account to make edits
1. click the edit button on this page in the docs (github will open to this page)
2. click the ✏️ icon to edit the page in github (a web based markdown editor will open)
3. make the change and preview (there's a preview tab near the top of the github web editor)
4. scroll to the bottom and create a new pull request for the change
5. submit PR and wait for it to be approved

Reply:
posted these instructions here:
https://docs.nearprotocol.com/docs/contribution/contribution-overview#content-creation